File: app/src/main/java/com/julio/api/network/APIClient.java
package com.julio.api.network;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class APIClient {

    public static Retrofit getClient() {
        String BASE_URL = "https://reqres.in";
        GsonConverterFactory factory = GsonConverterFactory.create();
        Retrofit.Builder builder = new Retrofit.Builder();
        builder.baseUrl(BASE_URL);
        builder.addConverterFactory(factory);
        Retrofit retrofit = builder.build();
        return retrofit;
    }
}

File: app/src/main/java/com/julio/api/network/APIClient.kt
package com.julio.api.network

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object APIClient {
    private val retrofit = Retrofit.Builder()
            .baseUrl("https://reqres.in")
            .addConverterFactory(GsonConverterFactory.create())
            .build()

    val services: EndPoint = retrofit.create(EndPoint::class.java)
}